staking/rust-toolchain

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.75.0"
```

Comment:
new versions of rust have a different Cargo.lock format but the version of `cargo-build-sbf` used in this repo is not compatible